### ISSUE TITLE
mistakes in code 

### ISSUE THREAD

**Issue by amiltonwong** [kentsyx/pointnet-pytorch#2]
Hi, @kentsyx ,

I found that [line96](https://github.com/kentsyx/pointnet-pytorch/blob/master/pointnet.py#L96) in pointnet.py should be modified as:
`return F.log_softmax(x, dim=1), trans`

according to [this reference](https://github.com/fxia22/pointnet.pytorch/blob/master/pointnet/model.py#L147).

**Comment by yunxiaoshi** [comment]
You are right! Fixed it
